vocabularies/geoadmin-feature-types.ttl

```diff
@@ -0,0 +1,69 @@
+    skos:broader <http://linked.data.gov.au/def/geoadminfeatures#AdministrativeFeature> ;
+    skos:definition "A Block is an administrative division of the State of Queensland that is used for tenure management. A Block is defined as five minutes of latitude by five minutes of longitude. Blocks are grouped by 1:1,000,000 map sheet areas, the names of which provide the containing Blocks with a four letter prefix (e.g. Clermont 1:1,000,000 provides CLER). Each sheet area contains 3456 Blocks, with the Blocks numbered between 1 and 3456 inclusive. A Block is divided into 25 Sub-Blocks."@en ;
+    skos:inScheme <http://linked.data.gov.au/def/geoadminfeatures> ;
+    skos:prefLabel "Block"@en .
+
+<http://linked.data.gov.au/def/geoadminfeatures#Permit> a skos:Concept ;
+    skos:broader <http://linked.data.gov.au/def/geoadminfeatures#AdministrativeFeature> ;
+    skos:definition "Permits are a license granted over a spatial region within the State of Queensland. Permits govern the activities which can be conducted upon them. Activities include exploration, development, and extraction of a specific commodity or commodities."@en ;
+    skos:inScheme <http://linked.data.gov.au/def/geoadminfeatures> ;
+    skos:prefLabel "Permit"@en .
+
+<http://linked.data.gov.au/def/geoadminfeatures#SubBlock> a skos:Concept ;
+    skos:broader [ ],
+        <http://linked.data.gov.au/def/geoadminfeatures#AdministrativeFeature> ;
+    skos:definition "A Sub-Block is an administrative sub-division of a Block. Each Block is divided into 25 Sub-Blocks. A Sub-Block is defined as one minute of latitude by one minute of longitude. The Sub-Blocks are prefixed with their corresponding Block name and are labelled from A to Z, excluding I. Labels are allocated from left to right, top to bottom, starting from the northwest corner."@en ;
+    skos:inScheme <http://linked.data.gov.au/def/geoadminfeatures> ;
+    skos:prefLabel "Sub-Block"@en .
+
+<http://linked.data.gov.au/def/geofeatures> skos:definition "This vocabulary lists types of geological features relevant to the duties of the Geological Survey of Queensland. It is automatically derived from GSQ's Geologic Features Ontology (http://linked.data.gov.au/def/geofeatures)"@en ;
+    skos:prefLabel "Geologic Feature Types"@en .
+
+<https://orcid.org/0000-0002-8742-7730> a owl:NamedIndividual,
+        sdo:Person ;
+    sdo:affiliation [ a sdo:Organization ;
+            sdo:name "SURROUND Australia Pty Ltd" ;
+            sdo:url <https://surroundaustralia.com> ] ;
+    sdo:email <mailto:nicholas.car@surroundaustralia.com> ;
+    sdo:name "Nicholas J. Car" .
+
+<http://linked.data.gov.au/org/gsq> a sdo:Organization ;
+    sdo:email <mailto:geological_info@dnrme.qld.gov.au> ;
+    sdo:name "Geological Survey of Queensland" ;
+    sdo:url <https://www.business.qld.gov.au/industries/mining-energy-water/resources/geoscience-information/gsq> .
+
+<http://linked.data.gov.au/def/geoadminfeatures> a owl:Ontology,
```

Comment:
There is no prefLabel for the conceptScheme

Reply:
Added: it was missing due to a fault in the auto-generation script (not updated properly from the GeoFeatures script)